README.md

```diff
@@ -6,7 +6,7 @@ Selfhosted runners do not come with the GH CLI out of the box. This action is an
 
 ```yaml
 - name: Install GH CLI
-  uses: dev-hanz-ops/install-gh-cli-action
+  uses: dev-hanz-ops/install-gh-cli-action@v0.1.0
   with:
-    gh-cli-version: 2.14.2 # optional, see action.yml for current default
```

Comment:
please keep the comment. if you want, you can update the default in https://github.com/dev-hanz-ops/install-gh-cli-action/blob/master/action.yml#L8 and I'll release it as v0.1.1